sources/editor/Stride.Editor/EditorGame/ContentLoader/LoaderReferenceManager.cs

```diff
@@ -14,7 +14,7 @@ namespace Stride.Editor.EditorGame.ContentLoader
 {
     public class LoaderReferenceManager
     {
-        private struct ReferenceAccessor
+        private readonly record struct ReferenceAccessor
```

Comment:
What does `record` bring here?

Reply:
Generation of the optimized equality comparer, there is a `List<ReferenceAccessor>.Contains` call lower in the same file

Reply:
Missed that one.